.github/workflows/test.yml

```diff
@@ -29,6 +29,7 @@ jobs:
     - name: Setup OS dependencies
       uses: MSP-Greg/setup-ruby-pkgs@v1
       with:
+        bundler: 2.2.15
```

Comment:
should not be necessary / prefer not to set since this will come back to fail in the future